lib/pattern.h

```diff
@@ -21,11 +34,9 @@ struct pattern_fmt {
 	const struct pattern_fmt_desc *desc;
 };
 
-int parse_and_fill_pattern(const char *in, unsigned int in_len,
-			   char *out, unsigned int out_len,
-			   const struct pattern_fmt_desc *fmt_desc,
-			   struct pattern_fmt *fmt,
-			   unsigned int *fmt_sz_out);
+int parse_and_fill_pattern_alloc(const char *in, unsigned int in_len,
+		char **out, const struct pattern_fmt_desc *fmt_desc,
```

Comment:
Is this an incorrect white space alignment or is this consistent with fio coding style when the function name is long?

Reply:
Hard to say. The two tab style is definitely preferred in Linux.

fio style seems to largely follow kernel style but doesn't seem to enforce line length at all.

I have found one place where this same style is used but many more places use overly long lines instead of wrapping.